src/features/prompts/server/prompt-router.ts

```diff
@@ -315,13 +319,15 @@ export const createPrompt = async ({
   prompt,
   isActive = true,
   createdBy,
+  config,
   prisma,
 }: {
   projectId: string;
   name: string;
   prompt: string;
   isActive?: boolean;
   createdBy: string;
+  config: JsonValue;
```

Comment:
i would define it as: config: z.infer<typeof jsonSchema>

Reply:
```suggestion
    config: z.infer<typeof jsonSchema>;
```